### PR TITLE
chore : applying integration20206 to qa-jcoin

### PR DESCRIPTION
**Pull request by atharvar28**
PR to apply `integration202006` branch for June Release Testing

### CODE REVIEW

File: qa-jcoin.planx-pla.net/manifest.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     "sidecar": {
       "cpu-limit": "1.0",
       "memory-limit": "256Mi",
-      "image": "quay.io/cdis/gen3fuse-sidecar:2020.05",
+      "image": "quay.io/cdis/gen3fuse-sidecar:integration202006",
```
Comment (themarcelor) [inline qa-jcoin.planx-pla.net/manifest.json:59]: This should be moved to `manifests/hatchery/hatchery.json`.

File: qa-jcoin.planx-pla.net/manifests/hatchery/hatchery.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
+{
+  "user-namespace": "jupyter-pods",
```
Comment (themarcelor) [inline qa-jcoin.planx-pla.net/manifests/hatchery/hatchery.json:2]: Maybe this should be:
`jupyter-pods-qa-jcoin`

File: qa-jcoin.planx-pla.net/manifest.json (hunk 1 above)
Comment (themarcelor) [reply qa-jcoin.planx-pla.net/manifest.json:59]: The entire hatchery block should be removed.

**Comment by paulineribeyre** [comment]
lgtm but i'll let the QA team do the final review

**Pull request merged**